README.md

```diff
@@ -1,2 +1,130 @@
+
+1. Fork or download the entire content-event-current-day repository. You will also need to run `npm install` and `bower install` to install the dependencies required for the the template. Bower is a package manager for Javascript libraries and frameworks. All third-party Javascript dependencies are listed in the bower.json file. To install Bower, run the following command in Terminal: `npm install -g bower`. Should you encounter any errors, try running the following command instead: `sudo npm install -g bower`.
+![diagram showing where to find the folder](img/src/img/readmeImage1.png)
+
+2. Inside the content-event-current-day folder you will find the src folder. You can modify the files inside the src folder to fit your needs (see below on how to modify specific elements).  
+![diagram showing file location](img/src/readmeImage2.png)
+
+3. Once any necessary modification have been made, a distribution folder containing all necessary files can be created using the following command while in the root diretory: `gulp build`. You can then host the newly created “dist” folder in rise vision storage or your web hosting service.  
+![diagram showing how to host folder](src/img/storageEventsReadme.png)
+
+4. Copy the link to the index.html file where you have it hosted and insert the url into a schedule. (you can also add this link to the url gadget within a presentation)  
+![diagram showing where to find link to index.html](img/readme-step4.jpg)
+
+##Directions to Modify the Event Theme:##
+
+####Changing the Background Image
+
+The background image for the theme can be changed by locating the following code within the style.css file within the css folder and modifying the path of the url to the image of your choice:
+
+```
+.wrapper:after {
+  ...
+  background-image: url("../img/city.jpeg");
+  ...
+```
+
+
+####Changing the Icon Images
+
+The images used for the icons in the theme are SVGs so that they will scale to fit different resolutions. To change the icons, open index.html, and locate the code for the specific icon within the section that you want to modify. For example, the code for the large calendar icon is found here:
+
+```  
+ <img src="img/calendar2.svg" class="date--calendarIcon" alt="calendar icon">
+```
+
+You can then change the src attribute to the path of the icon of your choice.
+
+
+####Changing The Text Content
+The text content is displayed through the use of the `rise-google-calendar` web component. Full details for this component can be found here # 
+In order for the component to access the data you need to create a public google calendar and ensure that all events you create are also public. The settings menu for the calendar is located in the top right corner of the calendar page. The button to make the event public is found here:
+![screenshot showing button to make event public](src/img/diagramPublicEvent.png)
+
+The next step is to locate the following code in the index.html file:
+
+```
+  <rise-google-calendar
```

Comment:
@pcsandford This is old code from `index.html` regarding the use of `rise-google-calendar`. Need to update to what is current.